client/my-sites/site-settings/manage-connection/disconnect-site-link.jsx

```diff
@@ -23,9 +25,7 @@ class DisconnectSiteLink extends Component {
 		dialogVisible: false,
 	};
 
-	handleClick = event => {
-		event.preventDefault();
```

Comment:
This call prevents that ugly `#` being appended to the URL when the disconnect link is clicked.

Maybe worth handling the `cursor: pointer` styling now or adding:

```js
if ( ! isEnabled( 'manage/site-settings/disconnect-flow' ) ) {
  event.preventDefault();
}
```

Reply:
I'll go for the quick fix. I'd really like to fix the cursor handling globally, but OTOH don't want to block this PR by it.

Reply:
58ef369